package.json

```diff
@@ -165,25 +165,25 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "bbc-a11y": "^2.4.1",
     "brotli-webpack-plugin": "^1.1.0",
-    "chalk": "^4.0.0",
+    "chalk": "^4.1.0",
     "chrome-launcher": "^0.13.3",
     "compression-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^6.0.2",
-    "cypress": "4.7.0",
+    "cypress": "4.8.0",
     "cypress-multi-reporters": "^1.4.0",
     "depcheck": "^0.9.2",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.5.0",
     "eslint": "^7.2.0",
-    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-airbnb": "^18.2.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-cypress": "^2.11.1",
-    "eslint-plugin-import": "^2.21.1",
+    "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-json": "^2.1.1",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-jsx-a11y": "6.2.3",
```

Comment:
Accidental removal of `^`?

Reply:
I locked this to that particalur version as it was breaking on of the tests, however with the latest version the tests now passes.